### DISCUSSION TITLE
"Experiments" feature triggering

### DISCUSSION THREAD

**Discussion by dupliaka** [Hyperfoil/Horreum#565] [Q&A] [answered]
Hello,

I was playing with "Experiments" feature to compare dataset provided by experimental branch of application with main as a baseline.
Looking to the code base - after setup experiment labels - it should trigger `ExperimentModal`. It seems to me that this modal should run the comparison but I didn't find the way how may I trigger this modal in the guide https://horreum.hyperfoil.io/.  

Also when I use labels I miss Jenkins plugin schema `urn:horreum:jenkins-plugin:0.1`that could be helpfull to filter datasets.

**Comment by johnaohara** [comment, marked as answer]
Hi @dupliaka 
 Once an Experiment Profile has been defined for a test;

![Screenshot from 2023-06-13 11-35-54](https://github.com/Hyperfoil/Horreum/assets/959822/3dfe0401-edb8-452e-8a90-9add3105631a)

An "Evaluate Experiment" button should appear of a dataset view for that test;

![Screenshot from 2023-06-13 11-33-26](https://github.com/Hyperfoil/Horreum/assets/959822/24dea1f2-877e-46fa-af29-e0ea3cc2934a)

Clicking on the "Evaluate Experiment" button, the result modal is displayed.

![Screenshot from 2023-06-13 11-33-39](https://github.com/Hyperfoil/Horreum/assets/959822/191fb675-0b9a-40e6-a293-0391b805b70c)

The guide needs updating to document how to set up and use experiments

`Also when I use labels I miss Jenkins plugin schema urn:horreum:jenkins-plugin:0.1 that could be helpfull to filter datasets.` - can you expand on this more? where is this an issue? which screen?

Thanks

**Reply by dupliaka** [reply]
It happens if I set up Labels for Experiment selector

![image](https://github.com/Hyperfoil/Horreum/assets/21653197/43b88d2e-c6d2-43d6-b254-6ec2f3dcbf69)

Despite the fact that `Build Url` lable exists in  `urn:horreum:jenkins-plugin:0.1` it does not show the schema

**Reply by johnaohara** [reply]
This is caused by "Build Url" label not being defined as a filtering label;

<img width="399" alt="Screenshot 2023-06-15 at 07 55 12" src="https://github.com/Hyperfoil/Horreum/assets/959822/fdcddc59-6f3d-4815-ad1f-d74585a6d3a7">

I'll open another issue, as a label that is neither filtering or metric is not much us, we should warn the user

**Reply by dupliaka** [reply]
I see now, thank you very much
